intro-HPC/ch_account.tex

```diff
@@ -515,7 +517,7 @@ \subsection{Adding multiple SSH public keys (optional)}
  \item Create a new public/private SSH key pair from Putty. Repeat the process described in section~\ref{sec:generate-key-pair}.
 \fi %ifwindows
  \item Go to \url{https://account.vscentrum.be/django/account/edit}
- \item Upload the new SSH public key using the \strong{Add public key} section.
+ \item Upload the new SSH public key using the \strong{Add public key} section. Make sure that your public key is actually saved, becasue a public key will be refused if it is too short, wrong type, or in a wrong format.
```

Comment:
typo `becasue` -> `because`
and please keep line length to 100-120 chars so it stays easy to reveiw diffs. not sure if `\item` is an issue here